assets/chef-recipes/cookbooks/mdbe_build/recipes/default.rb

```diff
@@ -0,0 +1,406 @@
+# frozen_string_literal: true
+
+case node[:platform]
+when 'debian'
+  package 'apt-utils'
+  package 'build-essential'
```

Comment:
You can put the list of packages into an array and then iterate over it.

The contents of the array should be sorted alphabetically and each should take 1 line.